week-1-ts-ecosystem/w01e04-light-me-up/src/LightBulb.ts

```diff
@@ -13,10 +13,23 @@ export class LightBulb {
     console.log("Lighting...");
   }
 
-  lightUpFor(seconds) {
+  lightUpFor(seconds: number, onError?: (e: Error) => void) {
```

Comment:
Bardzo fajnie @NiestrojMateusz 

Zastosowałeś jeden z 2 sposobów rozwiązania tego problemu - `callback`.
Drugim jest zapakowanie tego w `Promise` czyli mikrotask.

Jeśli chcesz możesz pokombinować z rozwiązaniem `promisified` 🎓 
Ale to jest jak najbardziej - ok.